docs/azure_arc_jumpstart/azure_arc_ml/aks/aks_blob_mnist_arm_template/_index.md

```diff
@@ -0,0 +1,238 @@
+
+## Deployment
+
+As mentioned, this deployment will leverage ARM templates. You will deploy a single template that will initiate the entire automation for this scenario.
+
+* The deployment is using the ARM template parameters file. Before initiating the deployment, edit the [_azuredeploy.parameters.json_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_ml_jumpstart/aks/arm_template/azuredeploy.parameters.json) file located in your local cloned repository folder. An example parameters file is located [here](https://github.com/microsoft/azure_arc/blob/main/azure_arc_ml_jumpstart/aks/arm_template/artifacts/azuredeploy.parameters.example.json).
+
+  * `sshRSAPublicKey` - Your SSH public key
+  * `spnClientId` - Your Azure service principal id
+  * `spnClientSecret` - Your Azure service principal secret
+  * `spnTenantId` - Your Azure tenant id
+  * `windowsAdminUsername` - Client Windows VM Administrator name
+  * `windowsAdminPassword` - Client Windows VM Password. Password must have 3 of the following: 1 lower case character, 1 upper case character, 1 number, and 1 special character. The value must be between 12 and 123 characters long.
+  * `myIpAddress` - Your local public IP address. This is used to allow remote RDP and SSH connections to the client Windows VM and AKS cluster.
+  * `logAnalyticsWorkspaceName` - Unique name for the deployment log analytics workspace.
+  * `kubernetesVersion` - AKS version
+  * `dnsPrefix` - AKS unique DNS prefix
+
+* To deploy the ARM template, navigate to the local cloned [deployment folder](https://github.com/microsoft/azure_arc/blob/main/azure_arc_ml_jumpstart/aks/arm_template) and run the below command:
+
+    ```shell
+    az group create --name <Name of the Azure resource group> --location <Azure Region>
+    az deployment group create \
+    --resource-group <Name of the Azure resource group> \
+    --name <The name of this deployment> \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_ml_jumpstart/aks/arm_template/azuredeploy.json \
+    --parameters <The *azuredeploy.parameters.json* parameters file location>
+    ```
+
+    > **Note: Make sure that you are using the same Azure resource group name as the one you've just used in the _azuredeploy.parameters.json_ file**
+
+    For example:
+
+    ```shell
+    az group create --name Arc-Data-Demo --location "East US"
+    az deployment group create \
+    --resource-group Arc-ML-Demo \
+    --name arcdata \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_ml_jumpstart/aks/arm_template/azuredeploy.json \
+    --parameters azuredeploy.parameters.json
+    --parameters templateBaseUrl="https://raw.githubusercontent.com/your--github--handle/azure_arc/main/azure_arc_ml_jumpstart/aks/arm_template/"
+    ```
+
+    > **Note: The deployment time for this scenario can take ~8-10min**
```

Comment:
Really?

Reply:
I was able to get an 8 minute deployment, the Bootstrap in ML is lighter than Data:
![8 min](https://raw.githubusercontent.com/microsoft/azure_arc/a34edef5b0e9c23ea135aa6775c9c53f695b49f5/docs/azure_arc_jumpstart/azure_arc_ml/aks/aks_blob_mnist_arm_template/03.png)

That being said, changed readme to "10-15min" to avoid confusions